README.md

```diff
@@ -62,32 +62,41 @@ pip install pyronear
 
 ### datasets
 
-Access all pyrovision datasets just like any `torchvision.datasets.VisionDataset`:
+Access out OpenFire dataset just like any `torchvision.datasets.VisionDataset`:
```

Comment:
not all for now, we need to do the same for the wildfire dataset !

Reply:
Indeed, just a little typo "Access out OpenFire..." -> "Access the OpenFire ..."

Reply:
Actually, perhaps don't modify this, the README was edited in another ongoing PR